README.md

```diff
@@ -31,13 +29,253 @@ _NB: There is no compatibility with Windows environments at this time._
 6. _(Optional)_ To run the UI with its own API, first copy `.env-ui.sample` as `.env-ui`. Please see the [configuration](#configuration) section for more details.
 7. _(Optional)_ Run `make run-ui` (or `make run-ui-arm` on Apple Silicon) to have the Web UI available on <http://localhost:8088> (in production mode). Its HTTP auth credentials are set in `.env-ui`. The UI API also has a frontend for API documentation (default <http://localhost:3002>).
 
-> If you want to run the UI app in development mode, i.e. with HMR enabled, please follow the steps in the [Development (UI)](#development-ui) section.
+#### Docker Guide Requirements
+
+- Unix-based operating system (e.g. Debian, Arch, Mac OS)
+- [Docker Engine](https://docs.docker.com/engine/) `1.27+`
+- Makefile toolchain `GNU Make 3.81`
+
+ℹ️ _**NOTE:** There is no compatibility with Windows environments at this time._
+
+To help expedite a lot of the docker commands, a lot has been abstracted using make commands with the `Makefile`.
+Included in each command are the docker commands to show what is being run.
```

Comment:
What is this trying to say exactly?

Reply:
Talking with some devs, that were coming from a Docker background, I added to ever make command the equivalent of what was happening as docker command.

This is present for every `make` command

_Example:_
```bash
make private_key=<YOUR_WALLET_PRIVATE_KEY> add-private-key;
# (Equivalent)
#   docker exec issuer-vault-1 vault write iden3/import/pbkey key_type=ethereum private_key=<YOUR_WALLET_PRIVATE_KEY>;
```

Also revised the text to be the following for better clarification:

> Included in each step are the equivalent docker commands to show what is being run.